toolchain/check/import_ref.cpp

```diff
@@ -1121,6 +1123,35 @@ static auto GetLocalReturnSlotPatternId(
            .index = param_pattern.index}));
 }
 
+static auto GetLocalCallParamsId(ImportContext& context,
```

Comment:
A comment would be useful here. In particular, it'd be good to mention that the caller is required to have called `LoadLocalPatternConstantIds` first to ensure the types of the call parameters have already been loaded.